cms/static/js/views/video/transcripts/utils.js

```diff
@@ -129,7 +130,10 @@ define(['jquery', 'underscore', 'jquery.ajaxQueue'], function($) {
                 match = link.pathname.match(/\/{1}([^\/]+)\.([^\/]+)$/);
                 if (match) {
                     cache[url] = {
-                        video: match[1],
+                        /* avoid too long video name, as it will be used as filename for video's transcript
+                        and a filename can not be more that 255 chars, limiting here to 100.
+                        */
+                        video: match[1].slice(0, maxLengthVideoName),
```

Comment:
Any future change can be placed in `maxLengthVideoName`. My point of view is, it will be an overkill to create a function to just call another function(`slice`). If you found it better to have a function anyway, I am open to create it.
Thanks